src/components/autocomplete/autocomplete.scss

```diff
@@ -110,9 +110,17 @@ md-autocomplete {
     padding: 0 15px;
     line-height: 40px;
     height: 40px;
+    position: absolute;
+  }
+  .md-autocomplete-placeholder {
+    color: #757575;
+    font-size: 14px;
+    line-height: 40px; // Matches md-autocomplete height.
```

Comment:
It looks like this class matches the dimensions of md-autocomplete, overall, not just the line-height. Update and relocate the comment to the class level to explain this?